backend/models/core/Entity.js

```diff
@@ -204,52 +204,112 @@ class Entity {
+    FROM entities
+    LEFT JOIN entity_types ON entities.entity_type_id = entity_types.id
+    LEFT JOIN attributes ON attributes.entity_type_id = entity_types.id
+    LEFT JOIN \`values\` ON values.entity_id = entities.id AND values.attribute_id = attributes.id
+    WHERE entity_types.slug = :entity_type_slug AND 
+    ${conditionType} = :entity_id ORDER BY attributes.\`order\``;
     
     const attributes = await db.executeStatement(entityIntrospectionSQL, [
-      {name: 'entity_type_id', value:{longValue: entity_type_id}},
+      {name: 'entity_type_slug', value:{stringValue: entity_type_slug}},
+      {name: 'entity_id', value:{[propertyType]: entity_id}},
     ]);
-    
+
+  
+
     const valueBatchParams = attributes.records.reduce((collection, record) => {
       const [
+        {longValue: entityId},
         {longValue: attributeId},
         {stringValue: attributeName},
         {stringValue: attributeType},
       ] = record;
-      
+       
       return [ 
         ...collection,
-        [
-          {name: 'entity_id', value: {longValue: entity_id}},
+        [ 
+          {name: 'entity_id', value: {longValue: entityId}},
           {name: 'attribute_id', value: {longValue: attributeId}},
             //Refactor to encapsulate type switch
-          {name: 'value_string', value: (attributeType == 'text' || attributeType == 'image' || attributeType == 'link') ? {stringValue: entries[attributeName]} : {isNull: true}},
-          {name: 'value_long_string', value:  attributeType == 'textarea' ? {stringValue: entries[attributeName]} : {isNull: true}},
-          {name: 'value_double', value:  attributeType == 'float' ? {doubleValue: entries[attributeName]} : {isNull: true}},
+          {name: 'value_string', value: (attributeType == 'text' || attributeType == 'image' || attributeType == 'link') && entries[attributeName] ? {stringValue: entries[attributeName]} : {isNull: true}},
+          {name: 'value_long_string', value:  attributeType == 'textarea' && entries[attributeName] ?  {stringValue: entries[attributeName]} : {isNull: true}},
+          {name: 'value_double', value:  attributeType == 'float' && entries[attributeName] ? {doubleValue: entries[attributeName]} : {isNull: true}},
         ]
       ]    
       
     }, []); 
-    
+
+    const getAllIdSQL = `SELECT \`values\`.attribute_id                 
+    FROM entities
+    LEFT JOIN entity_types ON entities.entity_type_id = entity_types.id
+    LEFT JOIN attributes ON attributes.entity_type_id = entity_types.id
+    LEFT JOIN \`values\` ON values.entity_id = entities.id AND values.attribute_id = attributes.id
+    WHERE entity_types.slug = :entity_type_slug AND 
+    ${conditionType} = :entity_id`;
+
+
+    const ids = await db.executeStatement(getAllIdSQL, [
+      {name: 'entity_id', value:{[propertyType]: entity_id}},
+      {name: 'entity_type_slug', value:{stringValue: entity_type_slug}},
+    ]);
+
+    console.log(valueBatchParams)
```

Comment:
You can now remove this line `(console.log(valueBatchParams)` if you don't need it.